plugins/ssr/ssrcommon.ml

```diff
@@ -1162,15 +1163,15 @@ let tacTYPEOF c = Goal.enter_one ~__LOC__ (fun g ->
 
 (** This tactic creates a partial proof realizing the introduction rule, but
     does not check anything. *)
-let unsafe_intro env decl b =
+let unsafe_intro env decl ~relevance b =
```

Comment:
Why not get the relevance from the goal?

Reply:
That's what we do in the caller

Reply:
It's a bit weird to do it this way given we're already in the tactic monad...

Reply:
I agree with @ppedrot, if it's possible to get the relevance here from the goal, there's no need to give callers a footgun to pass the wrong relevance

Reply:
There's no "callers", it's a helper for the function right next to it